Gravity/Gravity.NUnit.Integration/RSAPI_IntegrationTest.cs

```diff
@@ -0,0 +1,395 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using kCura.Relativity.Client;
+using NUnit.Framework;
+using Relativity.API;
+using Relativity.Test.Helpers;
+using Relativity.Test.Helpers.ServiceFactory.Extentions;
+using Relativity.Test.Helpers.SharedTestHelpers;
+using Relativity.Test.Helpers.WorkspaceHelpers;
+using Gravity.Test;
+using System.Configuration;
+using System.Diagnostics.Eventing.Reader;
+using Gravity.Base;
+using Gravity.Test.TestClasses;
+using kCura.Relativity.Client.DTOs;
+using System.Reflection;
+using Gravity.Test.Helpers;
+
+namespace Gravity.NUnit.Integration
+{
+    [TestFixture]
+    public class RSAPI_IntegrationTest
+    {
+        #region Variables
+
+        //this is set to true and should skip the creation of the database, installation of the testing application, and the deletion of the workspace.
+        //must point _workspaceId = valid workspace with application already installed in the Setup
+        private bool _debug = Convert.ToBoolean(ConfigurationManager.AppSettings["Debug"]);
```

Comment:
these underscores do not follow the naming conventions used in Gravity. I think it's time we took care of #18.

Reply:
Agreed.  I have a basis for coding standards that we will likely be recommending based on some work done at Relativity. Since I have not posted those yet, I will mage this chance to conform to existing for now.

Reply:
Do you have a .editorconfig file at Relativity? If not, I can create one with rules setup to be as consistent with existing code as possible.

Reply:
(We can always do a cleanup after we agree on one.)